README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
```

Comment:
very memorable link, for people who don't remember stuff often that link will stay on their heads for a year or two.

approved